src/components/MessageItemText/MessageItemText.tsx

```diff
@@ -3,13 +3,29 @@ import PropTypes from 'prop-types';
 import Typography from '@material-ui/core/Typography';
 
 interface Props {
+    id?: string;
     text: string;
 }
 
-export default function MessageItemText({ text }: Props) {
-    return <Typography variant='body1'>{text}</Typography>;
+/** Returns the text in proper typography so it is consistent with the current theme throughout the app
+ *  @category Component
+ *  @constructor MessageItemText
+ *  @param props
+ *  @param {string} props.id The id of the wrapping <span>
+ *  @param {string} props.text The text to return wrapped in \<Typography\>
+ */
+export default function MessageItemText({ id, text }: Props) {
+    return (
+        <span id={id}>
+            <Typography variant='body1'>{text}</Typography>
+        </span>
+    );
 }
 
+MessageItemText.defaultProps = {
+    id: 'MessageItemText',
+};
+
 MessageItemText.propTypes = {
     text: PropTypes.string.isRequired,
 };
```

Comment:
Mismatch between propTypes and defaultProps -- eslint should be giving you an error here?

```js
MessageItemText.propTypes = {
    text: PropTypes.string.isRequired,
    id: PropTypes.string
};
```

Reply:
Interesting, let me reinstall node_modules and try again. I kept getting an eslint error complaining as `id` is optional and it needed a default value

Reply:
`propType "id" is not required, but has no corresponding defaultProps declaration`

Is the error I get when I add `id` to `MessageItemText.propTypes `

Reply:
That's weird, we'll go over this in today's meeting then.